### ISSUE TITLE
Plex LXC fails - Setting up Hardware Acceleration

### ISSUE THREAD

**Issue by B08Z** [tteck/Proxmox#24]
✔  Setting Up Hardware Acceleration... 
/bin/chmod: cannot access '/dev/dri/*': No such file or directory

Running the script from web shell I get the above error when setting up plex. 
I am runnig Prox on NUC  i7 with Intel Quick GPU. 
ls pci output:
00:00.0 Host bridge: Intel Corporation Xeon E3-1200 v6/7th Gen Core Processor Host Bridge/DRAM Registers (rev 03)
00:02.0 VGA compatible controller: Intel Corporation Iris Plus Graphics 650 (rev 06)

Is there anything I need to do before running the script?

**Comment by osaether** [comment]
Have you enabled IOMMU in BIOS?

**Comment by osaether** [comment]
Try disabling IOMMU in BIOS.

**Comment by B08Z** [comment]
> Try disabling IOMMU in BIOS.

Thanks, I will have to wait to try that.

**Comment by kchiem** [comment]
I just used this script to set up plex on my NUC8i5, but hardware transcoding didn't work.  Googling, I found slightly different directions:

This script added into /etc/pve/lxc/###.conf:

`lxc.cgroup2.devices.allow: c 226:0 rwm`
`lxc.cgroup2.devices.allow: c 226:128 rwm`
`lxc.cgroup2.devices.allow: c 29:0 rwm`
`lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir`
`lxc.mount.entry: /dev/dri/renderD128 dev/renderD128 none bind,optional,create=file`

The last line didn't seem necessary, since the parent folder is already passed.  The other directions I saw said to pass:

`lxc.mount.entry: /dev/fb0 dev/fb0 none bind,optional,create=file`

.. instead.  That's what I did, and hw transcode works now.  Well, I also made one more change, and I'm not sure if it's necessary or not:

I noticed that in the base OS/proxmox, /dev/dri is owned by root:render.  The render group has gid 103.  In the LXC, gid 103 is the group syslog.  Thus, I edited /etc/group and added the plex user to the syslog group in the LXC.
